src/twisted/web/test/test_cgi.py

```diff
@@ -363,6 +372,45 @@ class CGIScriptTests(unittest.TestCase):
     """
     Tests for L{twcgi.CGIScript}.
     """
+    def tearDown(self):
```

Comment:
Thanks for implementing this test.  The `teardown()` and `startServer` methods have been copied from `class CGITests`, which is fine to get this test going.
Can you refactor these two methods into a new mixin class, and reuse the mixin in `CGITests` and `CGIScriptTests`.

Also, as you are finishing this off, can you run the tox command here: https://twistedmatrix.com/trac/wiki/TwistedDevelopment#GettingYourPatchAccepted

just to double check you are addressing any style issues before pushing to GitHub?